src/factory.js

```diff
@@ -68,6 +69,8 @@ export class Factory {
     }
     this.renderQ = [];
     this.stave = null; // current stave
+
+    this.StaveNote = this.StaveNote.bind(this);
```

Comment:
My main issue here is that it makes `StaveNote` a special method in the class, which this pattern is only used for the test. I'm open to having something like this here, but maybe in a separate PR where it applies across all factory  methods.